libnavigation-core/src/main/java/com/mapbox/navigation/core/telemetry/PhoneState.kt

```diff
@@ -0,0 +1,20 @@
+package com.mapbox.navigation.core.telemetry
+
+import android.content.Context
+import com.mapbox.android.telemetry.TelemetryUtils
+
+/**
+ * Class that will hold the current states of the device phone.
+ */
+internal data class PhoneState(val context: Context) {
```

Comment:
Same as above re: move to `libnavigation-util`.

Reply:
now all properties are inmutable, does it right?

Reply:
@RingerJK Not sure what you mean.

Reply:
when `PhoneState` is created, all properties stay inmutable.  

```
val phoneState = PhoneState(context)
val vl = phoneState.volumeLevel
```
`phoneState.volumeLevel` always returns the same value, even if volume level has been changed

Reply:
if it's right, than ok